Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Climate Modeling Alliance"]
 version = "0.9.2"
 
 [deps]
+CLIMAParameters = "6eacf6c3-8458-43b9-ae03-caf5306d3d53"
```

Comment:
Could we keep the CliMAParameters dependency only in tests?

Reply:
Yep, I forgot to remove it.